test/manifest-parser-tests.js

```diff
@@ -55,14 +56,31 @@ describe('Manifest Parser', function() {
     assert.equal(parsedManifest.value.background_color.value, undefined);
   });
 
+  describe('icon parsing', function() {
+    it('parses basic string', function() {
+      let parsedManifest = manifestParser('{"icons": [{"src": "192.png", "sizes": "192x192"}]}');
+      assert(!parsedManifest.debugString);
+      assert(!parsedManifest.value.icons.debugString);
+      assert(!parsedManifest.value.icons.value[0].value.sizes.debugString);
+      assert.equal(parsedManifest.value.icons.value.length, 1);
+    });
+
+    it('finds three icons in the stub manifest', function() {
+      let parsedManifest = manifestParser(JSON.stringify(manifestStub));
+      assert(!parsedManifest.debugString);
+      assert(!parsedManifest.value.icons.value[0].value.sizes.debugString);
+      assert.equal(parsedManifest.value.icons.value.length, 3);
+    });
+  });
```

Comment:
add a test for the case you added (icons with multiple spaces in the sizes string)?